source/_posts/2014-12-01-telemetry-initializers.markdown

```diff
@@ -171,4 +172,4 @@ RequestTelemetry was traced from
    at System.Web.Hosting.PipelineRuntime.ProcessRequestNotification(IntPtr rootedObjectsPointer, IntPtr nativeRequestContext, IntPtr moduleData, Int32 flags)
 
 ```
-Use telemetry initializers wisely for properties that changes from one data item to another. There are other mechanisms to set global properties that will have the same value for all data items in the process.
\ No newline at end of file
+Use telemetry initializers wisely for properties that changes from one data item to another. There are other mechanisms to set global properties that will have the same value for all data items in the process.
```

Comment:
"There are other mechanisms" - It would be great if you can point to them - this sentence leaves the reader wondering what are those other mechanisms